tests/containers/volumes.pm

```diff
@@ -43,7 +43,7 @@ sub run {
     assert_script_run("mkdir -p $test_dir");
 
     # Create Dockerfile with VOLUME defined
-    assert_script_run("echo -e 'FROM busybox\nVOLUME /$test_dir' > $test_dir/Dockerfile");
+    assert_script_run('echo -e "FROM busybox\nVOLUME /$test_dir" > $test_dir/Dockerfile');
```

Comment:
This is wrong, here `$test_dir` will not be replaced by the variable content but remain `$test_dir` in the `Dockerfile`!

Reply:
Fixed.